src/metabase/integrations/common.clj

```diff
@@ -51,3 +55,12 @@
         (db/insert! PermissionsGroupMembership :group_id id, :user_id user-id)
         (catch Throwable e
           (log/error e (trs "Error adding User {0} to Group {1}" user-id id)))))))
+
+#_{:clj-kondo/ignore [:missing-docstring]}
```

Comment:
Does clj-kondo generate a warning without this line? I added a new multi-setting recently (`audit-max-retention-days`) and didn't get any missing docstring warnings so you might be fine without it.